yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
are you using yarn or npm - there should be only yarn.lock or package-lock.json

Reply:
so, should I delete yarn.lock??

Reply:
Alright I deleted packag-lock.json and everything was ok, its gone